.github/release.yml

```diff
@@ -0,0 +1,8 @@
+changelog:
+  categories:
+    - title: 🛠 Dependency Upgrades 
+      labels:
+        - dependencies
+    - title: ⭐ New Features
+      labels:
+        - "*"
```

Comment:
Why do we need this file?

Reply:
It's for the github release page. See example here
https://github.com/MelleD/pm-index-card/releases/tag/0.2.0

Reply:
Not sure if this is relevant to fixing the moon phase integration?